es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super();
+    this.width = width;
+    this.area = width ** 2;
+    this.perimeter = width * 4;
```

Comment:
Instead of defining these directly, you should be taking advantage of the `super` keyword.